app.js

```diff
@@ -31,12 +31,16 @@ app.use(session({
 app.use(express.static(path.join(__dirname, 'public')))
 
 // Tell Express to use our routers we've made.
-app.use('/', routers.web.router)
-app.use('/', routers.login.router)
-app.use('/', routers.signup.router)
-app.use('/', routers.dashboard.router)
-app.use('/', routers.profile.router)
-app.use('/', routers.error.router)
+app.use('/', routers.users.router)
+app.use('/', routers.submissions.router)
+app.use('/', routers.games.router)
+app.use('/', routers.signin.router)
+// app.use('/', routers.web.router)
```

Comment:
Do we want to just delete these and add them back as needed? Will look a bit cleaner then commenting them out.